jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-list-view.xml

```diff
@@ -40,8 +40,8 @@
         <dataLoadCoordinator auto="true"/>
     </facets>
     <layout spacing="true">
-        <split id="split" orientation="HORIZONTAL" width="100%" minHeight="25em" height="100%">
-            <vbox id="categoryBox" margin="false" padding="false" height="100%" expand="categoriesGrid">
+        <split id="split" orientation="HORIZONTAL" width="100%" minHeight="25em" height="100%" themeNames="splitter-spacing">
```

Comment:
This line is beyond the 120 character limit
Please break the line

![image](https://github.com/user-attachments/assets/844ae32e-b381-4d22-9bfc-a31dccf8ec6b)